plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaClientsModule.java

```diff
@@ -38,19 +40,30 @@ private void installClientModule(SecurityProtocol securityProtocol, Module modul
     {
         install(conditionalModule(
                 KafkaSecurityConfig.class,
-                config -> config.getSecurityProtocol().equals(securityProtocol),
+                config -> config.getSecurityProtocol().orElse(null) == securityProtocol,
```

Comment:
Can we use `Optional#filter(SecurityProtocol::equals).isPresent()`

Reply:
Applying this fails `TestKafkaPlugin` tests since `securityProtocol` is nullable and calling `security::equals` throws `NullPointerException`.